app/views/users/sign_in.html.haml

```diff
@@ -3,7 +3,7 @@
     .col-sm-10.col-sm-offset-1
       %form.sign_in
```

Comment:
You should always use `form_tag`. That way xss protection is present in your form.